### ISSUE TITLE
Enhancment - Touch Slash

### ISSUE THREAD

**Issue by Anish-Agnihotri** [Continuities/adarkroom#209]
For touch support why not have a slash and stab feature whereas : 

A slide on screen is a slash
A tap on screen is a stab

Could this be implemented for attacking?

**Comment by skj3gg** [comment]
While a novel idea, I have no interest in developing `stab`, `swing`, `thrust`, `slash`, `shoot`, `blast`, `tangle`, or `eat meat` touch movement controls (let alone figure out the difference between `swing` and `slash` UX). If someone is extremely intrepid, I would review a PR though

**Comment by Continuities** [comment]
Yeah, I think this is a cool idea. The only problem I can see (and it's a
pretty big one) is how to communicate to the player which gesture triggers
each attack. There are a lot of weapons, and you'd need a different gesture
for each one. If someone can solve this problem, I'm all for it.

**Comment by Anish-Agnihotri** [comment]
Oh Alright, I'll try to figure out the gestures. Thanks

**Comment by gitupofftheflooranddosomework** [comment]
this would be crazily complicated and not worth it, this should be closed
